plugins/base/src/test/kotlin/parsers/JavadocParserTest.kt

```diff
@@ -47,4 +60,51 @@ class JavadocParserTest : BaseAbstractTest() {
             assertEquals("content being refreshed, which can be a result of invalidation, refresh that may contain content updates, or the initial load.", docs.trimEnd())
         }
     }
+
+    @Test
+    fun `code tag`() {
+        val source = """
+            |/src/main/kotlin/test/Test.java
+            |package example
+            |
+            | /**
+            | * Identifies calls to {@code assertThat}.
+            | *
+            | * {@code
+            | * Set<String> s;
+            | * System.out.println("s1 = " + s);
+            | * }
+            | * <pre>{@code
+            | * Set<String> s2;
+            | * System.out
+            | *         .println("s2 = " + s2);
+            | * }</pre>
+            | * 
+            | */
+            | public class Test {}
+            """.trimIndent()
+        testInline(
+            source,
+            configuration,
+        ) {
+            documentablesCreationStage = { modules ->
+                val docs = modules.first().packages.first().classlikes.single().documentation.first().value
+                val root = docs.children.first().root
+
+                kotlin.test.assertEquals(
+                    listOf(
+                        Text(body = "Identifies calls to "),
+                        CodeInline(children = listOf(Text(body = "assertThat"))),
+                        Text(body = ". "),
+                        CodeInline(children = listOf(Text(body = "\nSet<String> s;\nSystem.out.println(\"s1 = \" + s);\n")))
```

Comment:
Is this test correct? Original javadoc for JDK 11 outputs this:

![image](https://user-images.githubusercontent.com/15652452/128863201-05b5deca-59c9-47af-98dd-3bc8e15fe37d.png)

Reply:
My screenshot (Original javadoc from  adopt JDK 11):
![image](https://user-images.githubusercontent.com/9636937/128927626-93ecae1e-16c2-4c1e-b707-6e6c3a9869f5.png)
It produces following html code:
```html
<div class="block">Identifies calls to <code>assertThat</code>.

 <code>
 Set&lt;String&gt; s;
 System.out.println("s1 = " + s);
 </code>
 <pre><code>
 Set&lt;String&gt; s2;
 System.out
         .println("s2 = " + s2);
 </code></pre></div>
```
---
Dokka produces:
![image](https://user-images.githubusercontent.com/9636937/128928584-184a08b4-dfd5-49e7-a61e-5d6c59876a17.png)

```html
<p class="paragraph">Identifies calls to <code>assertThat</code>. <code>
Set&lt;String&gt; s;
System.out.println(&quot;s1 = &quot; + s);
</code></p><div class="sample-container"><code class="" theme="idea"><pre>
Set&lt;String&gt; s2;
System.out
        .println(&quot;s2 = &quot; + s2);
</pre></code>
```

Dokka with `javadoc` plugin:
![image](https://user-images.githubusercontent.com/9636937/129008874-4e8ab719-f138-4d2c-a9a7-0b5fcaa49ab3.png)

```html
                    <div class="block"><p>Identifies calls to <code>assertThat</code>. <code>
Set&lt;String&gt; s;
System.out.println("s1 = " + s);
</code></p><pre><code>
Set&lt;String&gt; s2;
System.out
        .println("s2 = " + s2);
</code></pre></div>
```